tsl/src/compression/create.c

```diff
@@ -621,15 +621,6 @@ create_compress_chunk_table(Hypertable *compress_ht, Chunk *src_chunk)
 	if (!OidIsValid(compress_chunk->table_id))
 		elog(ERROR, "could not create compressed chunk table");
 
```

Comment:
```suggestion
Constraints and triggers are not created on the PG chunk table. Caller is expected to do this explicitly.
```
would be good to add a comment here or as a comment to the function.